task_sdk/src/airflow/sdk/execution_time/task_runner.py

```diff
@@ -693,6 +694,16 @@ def run(
     return state, msg, error
 
 
+def _handle_current_task_success(context, ti) -> tuple[SucceedTask, TerminalTIState]:
```

Comment:
Is this called anywhere?

Reply:
Yes - for marking both regular tasks and skipping tasks, as successful. It's either that or the nested "try...except" from previous time :)